src/main/java/seedu/address/logic/commands/delivery/DeliveryListCommand.java

```diff
@@ -22,6 +22,7 @@
 public class DeliveryListCommand extends DeliveryCommand {
     public static final String COMMAND_WORD = DeliveryCommand.COMMAND_WORD + " " + "list";
     public static final String MESSAGE_SUCCESS = "Listed all Deliveries";
+    public static final String MESSAGE_EMPTY = "There are no deliveries!";
```

Comment:
In line with the customer empty message, would it be better to be `There are currently no deliveries!`

Reply:
I thought of that but it may imply that the user has absolutely no deliveries? Because the list might show that there are no deliveries because of the filters applied but there are still deliveries.

Reply:
Sorry I realised that this is list, then would it be more apt to say listed 0 deliveries or no deliveries to be listed